utilities/build_envs.sh

```diff
@@ -24,10 +24,14 @@ case "${ARCH?}" in
         JULIA_CPU_TARGETS+=(
             # Absolute base x86_64 feature set
             "generic"
+            # Architecture implemented by Apple's Rosetta 2 translation layer
+            "westmere,aes"
```

Comment:
I am not convinced that we should add this. People should just be using the native binaries at this point.

Reply:
Unfortunately, that’s not always possible due to other dependencies or (as in my own case) due to needing to test packages under Rosetta 2 as a deliberate use case.  I can currently get away with using the existing x86_64 builds but need to constantly specify the target string in both the environment variable and command-line flag forms; omitting either one causes crashing with either illegal instructions or internal errors in the Julia binary…and inspecting the package dylibs with elfx86exts shows me there’s a lot of performance being left on the table.

Reply:
> omitting either one causes crashing with either illegal instructions or internal errors in the Julia binary

That would be worth bug reports, but new targets take up significant space and and this needs to be broadly useful.

Reply:
@vchuravy Understood; I'll remove it.